optimum/utils/dummy_diffusers_objects.py

```diff
@@ -68,3 +68,14 @@ def __init__(self, *args, **kwargs):
     @classmethod
     def from_pretrained(cls, *args, **kwargs):
         requires_backends(cls, ["diffusers"])
+
+
+class ORTLatentConsistencyModelPipeline(metaclass=DummyObject):
```

Comment:
Is this essentially putting a requirement on diffusers==v0.22 (next release)? Could it make sense to https://github.com/huggingface/optimum/blob/f223f5e330b678767c76489b3c30154d6959c80c/optimum/utils/dummy_bettertransformer_objects.py#L19 ? Just wondering

Reply:
No it doesn't, however we'll need diffusers>=v0.22.0 for the tests (so that we can compare the ORT pipeline with the diffusers `LatentConsistencyModelPipeline` pipeline)